07.ls_object/file_stat.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'etc'
+
+class FileStat
+  attr_reader :blocks, :file_mode, :nlink, :user_name, :group_name, :size, :mtime, :name
+
+  def initialize(file_name)
+    file_stat = File.stat(file_name)
+
+    @blocks = file_stat.blocks
+    @file_mode = FileMode.new(file_name).build
```

Comment:
```suggestion
    @file_mode = FileMode.new(file_name)
```

であれば自然ですが、`build`が付くと「？」となってしまいますね。

https://bootcamp.fjord.jp/pages/readable-variable-name

変数名かクラス名かメソッド名か、何かの命名をミスってるかもしれません

Reply:
https://github.com/icchankun/ruby-practices/pull/11#discussion_r1574139689
こちらは、上のコメントに対する修正コミットで対応しています！